Doc/library/gzip.rst

```diff
@@ -211,3 +211,38 @@ Example of how to GZIP compress a binary string::
       The basic data compression module needed to support the :program:`gzip` file
       format.
 
+Command Line Interface
+----------------------
+
+The :mod:`gzip` module provides a simple command line interface to compress or
+uncompress files.
+
+Once executed the :mod:`gzip` module keeps the input file(s).
+
+.. versionadded:: 3.8
```

Comment:
The command line interface was present previous versions too. Maybe this can be changed to `versionchanged` to indicate argparse is used instead of manual parsing?